providers/base/bin/pipewire_utils.py

```diff
@@ -533,6 +603,11 @@ def _args_parsing(self, args=sys.argv[1:]):
                 "-t", "--type", type=str, required=True,
                 help="VIDEO or AUDIO")
 
+        # Add parser for show current status function
+        parser_show = subparsers.add_parser(
+                'status',
+                help='show pipewire current status from wpctl status')
+
```

Comment:
Instead of having a function that prints the modified output of wpctl status, I think it would be easier to just compare the output of two raw statuses, that way, you won't have to modify much of the checkbox job and you could get rid of the diff from the CLI:
```
        # Add parser to compare to statuses
        parser_status = subparsers.add_parser(
                'compare_status',
                help='compare the output from two wpctl status')
        parser_status.add_argument(
                "-s1", "--status1", type=str,
                help="path to the second output of wpctl status")
        parser_status.add_argument(
                "-s2", "--status1", type=str,
                help="path to the second output of wpctl status")
```

In the job you would have something like this:

```
plugin: shell
category_id: com.canonical.plainbox::suspend
id: suspend/audio_before_suspend
...
command:
  if check_audio_daemon.sh ; then
    wpctl status > "$PLAINBOX_SESSION_SHARE"/audio_settings_before_suspend
```

```
plugin: shell
category_id: com.canonical.plainbox::suspend
id: suspend/audio_after_suspend
...
command:
  if check_audio_daemon.sh ; then
      wpctl status > "$PLAINBOX_SESSION_SHARE"/audio_settings_after_suspend
      pipewire_utils.py compare_status -s1 "$PLAINBOX_SESSION_SHARE"/audio_settings_after_suspend -s2 "$PLAINBOX_SESSION_SHARE"/audio_settings_after_suspend
  else
      audio_settings.py store --file="$PLAINBOX_SESSION_SHARE"/audio_settings_after_suspend
      diff "$PLAINBOX_SESSION_SHARE"/audio_settings_before_suspend "$PLAINBOX_SESSION_SHARE"/audio_settings_after_suspend
  fi
```

Reply:
This will be in another PR.